src/user/index.tsx

```diff
@@ -53,15 +61,18 @@ const User = () => {
     });
   }, [data]);
 
-  useEffect(() => {
-    fetchData();
-  }, []);
-
   const handlemoveEditPage = () => {
     navigate(`/userEdit/${id}`);
   };
   const totalLikes =
     userInfo.posts && userInfo.posts.reduce((acc, cur: Pick<IPost, 'likes'>) => acc + cur.likes.length, 0);
+
+  const followbutton =
+    userFollow.followers && userFollow.followers.includes(myUserId as string) ? (
+      <button>팔로잉</button>
+    ) : (
+      <button>팔로우</button>
+    );
```

Comment:
h4;
button 태그가 반복되어서 이렇게 정리하셔도 좋을 것 같습니다!
```suggestion
  const followbutton = <button>{userFollow?.followers?.includes(myUserId as string) ? "팔로잉" : "팔로우"}</button>
```